docs/deployment.rst

```diff
@@ -156,5 +160,5 @@ and no other interpreter or apllication has to be deployed.
 See also
 --------
 
-* `SSH Daemon manpage<http://www.openbsd.org/cgi-bin/man.cgi/OpenBSD-current/man8/sshd.8>`_
-* `Ansible<http://docs.ansible.com>`_
+* `SSH Daemon manpage <http://www.openbsd.org/cgi-bin/man.cgi/OpenBSD-current/man8/sshd.8>`_
+* `Ansible <http://docs.ansible.com>`_
```

Comment:
Ok I see, but the makefile could check if something is missing and report a more useful error than this. I expect normally when a makefile is invoked it automatically fixes all necessary dependencies.